### DISCUSSION TITLE
Is it possible to send structured logs via http to the agent ?

### DISCUSSION THREAD

**Discussion by PawOrmstrupMadsen** [netdata/netdata#19200] [Q&A] [answered]
Hi Netdata

We are looking into your solution, and are wondering if we can replace our PRTG monitoring "platform" and our Falcon Logscale Solution with Netdata.

Regarding the Metrics part that is handled in PRTG, it looks like a very solid solution from you, so not many questions on this part.

But regarding application logging we are using a Serilog Splunk Sink to send our application logs from a "Legacy Enterprise ECM solution (mostly .NET code)" into Logscale and a Humio (old name for LogScale) Sink for our self-developed solutions (.NET code), and that works very well for us - are there any endpoints in the Netdata Agent that can grab json structured logs being sent via http calls ?

if not - would it be possible to implement such an ingest endpoint, or is this not "best practice" for the agent?

We would like to avoid having to write logs to disk so that the Netdata agent can pick them up again.

This is all for the Windows Agent, but i guess that is irrelevant since such an ingest endpoint would be universal for applications to send logs to Netdata Agent.

for details on what LogScale has for Ingest endpoints see here:
https://library.humio.com/falcon-logscale-cloud/endpoints.html

For details on the Serilog Sinks we are currently using see here:
https://github.com/ThomasWillumsen/Serilog.Sinks.Humio
https://github.com/serilog-contrib/serilog-sinks-splunk

I hope this makes sense to ask for, i tried finding the answer from your docs, but have not found an answer yet.

Best Regards
Paw from Nobly A/S

**Comment by ilyam8** [comment, marked as answer]
Hi, @PawOrmstrupMadsen.

> We would like to avoid having to write logs to disk so that the Netdata agent can pick them up again.

Unfortunately, that is not possible. Netdata doesn't store or ingest logs directly. Instead, it reads system metrics from various sources (like journald or Windows Event Log), and allows viewing, exploring, and analyzing them.
